ConfigurationSystem/Service/ConfigurationHandler.py

```diff
@@ -75,6 +75,16 @@ def export_commitNewData(self, sData):
 
     return res
 
+  types_forceGlobalConfigurationUpdate = []
+
+  def export_forceGlobalConfigurationUpdate(self):
```

Comment:
Don't you want to hardcode permissions here ? Or at least set them in the template as diracAdmin, and leave a note in the wiki ?

Reply:
Yes, adding CS_ADMINISTRATOR default privelege